test/utils.js

```diff
@@ -24,13 +24,22 @@ var utils = {
         server.connection({port: options.port || 9100});
         return new Promise((resolve) => {
             server.register([
-                {register: require('../'), options: {adapter: adapter({mongodbUrl: 'mongodb://localhost/test'})}},
+                {
+                    register: require('../'),
+                    options: {
+                        adapter: adapter({mongodbUrl: 'mongodb://localhost/test'}),
+                        oplogConnectionString: 'mongodb://127.0.0.1:27017/local?slaveOk=true'
```

Comment:
This has to be an adapter options rather than a plugin option. Oplog is mongoDB only.